Makefile

```diff
@@ -38,11 +38,13 @@ MISSPELL=misspell -error
 MISSPELL_CORRECTION=misspell -w
 
 BUILD_INFO_IMPORT_PATH=github.com/signalfx/splunk-otel-collector/internal/version
+BUILD_INFO_IMPORT_PATH_TESTS=github.com/signalfx/splunk-otel-collector/tests/internal/version
```

Comment:
Need to use introduce another `version` package to avoid cross modules dependency. The root module already depends on `tests` module, so we cannot make `tests` module depend on the root module. 

I don't think we need `tests` module at all, but that can be handled separately.

Reply:
We could break out testutils since I think only the available port finder is shared but that can be done in another change that separates its tests from the actual integration runs.

Reply:
The `testutils` has to be put to another new module in that case.  Do we need `tests` module to reduce the build side?